lib/enhancements/edition.rb

```diff
@@ -140,4 +140,13 @@ def artefact
   def self.convertible_formats
     Artefact::FORMATS_BY_DEFAULT_OWNING_APP["publisher"] - ["local_transaction"]
   end
+
+  def fact_check_id
```

Comment:
@brenetic mentioned this was inspired by SecureRandom.uuid's implementation? Would be nice to leave a comment about that in the commit message, as this isn't the easiest code to understand at a glance